backend/canisters/proposals_bot/impl/src/jobs/retrieve_proposals.rs

```diff
@@ -184,11 +184,11 @@ fn handle_proposals_response<R: RawProposal>(governance_canister_id: CanisterId,
                     .take_newly_decided_user_submitted_proposals(governance_canister_id);
 
                 let now = state.env.now();
-                if let Some(ns) = state.data.nervous_systems.get(&governance_canister_id) {
-                    let ledger_canister_id = ns.ledger_canister_id();
-                    let amount = ns.proposal_rejection_fee().into();
-                    let fee = ns.transaction_fee().into();
-                    for proposal in decided_user_submitted_proposals {
+                for proposal in decided_user_submitted_proposals {
```

Comment:
I just moved the iteration over `decided_user_submitted_proposals` higher since it will be empty in 99.9% of cases.